src/main/java/net/minestom/server/extras/bungee/BungeeCordProxy.java

```diff
@@ -32,6 +51,18 @@ public static boolean isEnabled() {
         return enabled;
     }
 
+    public static void setBungeeGuardTokens(@Nullable Set<String> tokens) {
+        bungeeGuardTokens = tokens;
+    }
+
+    public static boolean isBungeeGuardEnabled() {
+        return bungeeGuardTokens != null;
+    }
+
+    public static int getMaxLength() {
+        return isEnabled() ? (isBungeeGuardEnabled() ? 2500 : Short.MAX_VALUE) : 255;
```

Comment:
might want to put a comment here so minestom devs know this is trying to emulate how the bungeeguard plugin works